packages/components/src/local/form-elements/new-message/index.stories-inc.tsx

```diff
@@ -0,0 +1,70 @@
+import { MessageDetails } from '@serge/custom-types'
+import { P9Mock, planningMessageTemplatesMock } from '@serge/mocks'
+import { noop } from 'lodash'
+import React from 'react'
+import NewMessage2 from './index'
+import docs from './README.md'
+
+const wrapper: React.FC = (storyFn: any) => <div style={{ height: '700px' }}>{storyFn()}</div>
+
+export default {
+  title: 'local/form-elements/NewMessage2',
+  component: NewMessage2,
+  decorators: [wrapper],
+  parameters: {
+    readme: {
+      content: docs
+    },
+    options: {
+      showPanel: true
+    },
+    controls: {
+      expanded: true
+    }
+  },
+  argTypes: {
+    privateMessage: {
+      description: 'Provide private message (umpire force)',
+      control: {
+        type: 'boolean'
+      }
+    },
+    orderableChannel: {
+      description: 'Whether messages are shown in order',
+      control: {
+        type: 'boolean'
+      }
+    },
+    confirmCancel: {
+      description: 'Whether player has to confirm cancelling a message',
+      control: {
+        type: 'boolean'
+      }
+    }
+  }
+}
+
+export const saveMessage = (dbName: string, details: MessageDetails, message: object): Function => (): Function => {
```

Comment:
Aah, @lebaphi - I could do with help from you here, please.  I can't handle the T/S to pass `saveMessage` in the props below. Could you help me out, and get it working please?

Reply:
sure @IanMayo